core/ExceptionHandler.php

```diff
@@ -41,6 +44,8 @@ public static function handleException($exception)
      */
     public static function dieWithCliError($exception)
     {
+        self::logException($exception);
```

Comment:
Likely not too important but just realising that `console` is doing eg `Piwik\ExceptionHandler::setUp();` (and also `Piwik\ErrorHandler::registerErrorHandler();`) before the environment is loaded and container is built. So there could be under circumstances a problem where a message is maybe logged but the container is not ready yet. Not sure what would happen and likely the chances are maybe low there is an error between registering error handler and loading environment...

Reply:
actually stupid of me... I'm just seeing you catch an exception there anyway. Forget the comment :)